parquet-encoding/src/main/java/org/apache/parquet/column/values/bitpacking/BytePacker.java

```diff
@@ -105,4 +116,26 @@ public void unpack8Values(final byte[] input, final int inPos, final int[] outpu
   public void unpack32Values(byte[] input, int inPos, int[] output, int outPos) {
     unpack32Values(ByteBuffer.wrap(input), inPos, output, outPos);
   }
+
+  /**
+   * unpack bitWidth bytes from input at inPos into {unpackCount} values in output at outPos using Java Vector API.
+   * @param input the input bytes
+   * @param inPos where to read from in input
+   * @param output the output values
+   * @param outPos where to write to in output
+   */
+  public void unpackValuesVector(final byte[] input, final int inPos, final int[] output, final int outPos) {
```

Comment:
Do we have a standard norm to define a function when it leverages the Vector API? Does it sound good to rename it to `unpackValuesUsingVector` or `vectorUnpackValues`? The current name is misleading.

Reply:
@wgtmac I have reviewed the Java Vector Module name norm, and renamed related classed and methods, unpackValuesUsingVector looks good, so I used it.